scripts/scripts.js

```diff
@@ -209,13 +195,39 @@ async function loadEager(doc) {
   }
 }
 
+async function toolReady() {
```

Comment:
this prevents occasional CLS caused by tool loading. Basically plays lazy until the tool is read. will go back later and add this to other tools.